src/client.js

```diff
@@ -860,22 +862,36 @@ MatrixClient.prototype.disableKeyBackup = function() {
  * Set up the data required to create a new backup version.  The backup version
  * will not be created and enabled until createKeyBackupVersion is called.
  *
+ * @param {string} password Passphrase string that can be entered by the user
+ *     when restoring the backup as an alternative to entering the recovery key.
+ *     Optional.
+ *
  * @returns {object} Object that can be passed to createKeyBackupVersion and
```

Comment:
this now returns a promise, not an object.

Reply:
Hmm, I was assuming the promise was implicit for async function, but the convention seem to be to put it explicitly.